src/Date/DatePickerInput.shared.tsx

```diff
@@ -21,6 +21,7 @@ export type DatePickerInputProps = {
   endYear?: number
   onChangeText?: (text: string | undefined) => void
   inputEnabled?: boolean
+  inputButton?: React.ReactNode
```

Comment:
If this is a new prop I think we should consider adding it to the [docs](https://web-ridge.github.io/react-native-paper-dates/docs/category/date-picker) so it's visible to others who may want to use this.

Reply:
> If this is a new developer facing prop, I think we should consider adding it to the [docs](https://web-ridge.github.io/react-native-paper-dates/docs/category/date-picker) so it's visible to others who may want to use this.

Thanks for the review.
I have added it only to share the information in another component, but in those props the user cannot use it because it is already defined by you indicating the calendar.
Actually I tried to follow what was done but even I think I can delete all that code where it assigns the inputButtons and leave it only in the React Native Paper Input and it would be cleaner.

I didn't want to do it before because I thought they were "big" changes and maybe they don't want to do them, but if you want I try and if everything goes well I think it would be better.